plasmapy/particles/particle_class.py

```diff
@@ -273,51 +273,16 @@ class Particle:
 
     """
 
-    def __init__(
-            self,
+  
+   ##############
+
```

Comment:
Delete lines...[PEP8: there should be a single blank line between methods](https://www.python.org/dev/peps/pep-0008/#blank-lines)